data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,346 @@
+			label tellme
+			`	"We have a very interesting proposition for you: should you agree to bring supplies to our brethren, the Unfettered Hai, we will offer you a ship, for a modest price of 6.5 million credits. This is a very useful ship, and you should be able to repay this quite easily by running more of the same kind of jobs. It is indeed appropriate for bringing this important food cargo safely to destination, and even defend yourself, if need be. You see, these supplies are needed on short notice, on <destination>, and we lack the time and personnel to bring them."`
+			`	They take you into the cave, finally showing the ship they were referring to. "This is a mighty relic of the past, from the times our Empire used to control the stars; a Ladybug!`
+			scene "thumbnail/hai ladybug"
+			`	"Of course, it has some... drawbacks, compared to its golden time, and don't expect the best price in a shipyard either. Nevertheless it is a fine vessel, capable of safely taking a respectable amount of cargo across large distances! So, what do you say, <last>? This is the opportunity of a lifetime, as they simply don't make ships like these anymore!"`
+			choice
+				`	"I will take this deal."`
+					goto take
+				`	"I am not interest in this deal."`
+			`	"A shame, this probably means we will not meet again," they say, before leaving, as discreetly as they came.`
+				decline
+			label take
+			action
+				give ship "Ancient Ladybug" "Good Omen"
+				fine 6500000
+				log "Accepted a somewhat shady deal for a old Hai ship, a Ladybug, that seems to still be in good functioning state."
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go trough official means for this, we will have to present it as a fine. Worry not, we will make it something of no consequence, like an old Korath forbidden tech you would not have known about, that we had to confiscate."`
+			`	You quickly realize they are not asking you for your opinion at all. In fact, you notice the fine has already been issued before you could say anything. "Oh, and you should not change its colors, you'll understand why later."`
+				accept
+
+mission "Hai Legacy 2"
+	shipyard
+	landing
+	name "Young but not that young"
+	description "Use the newly acquired old Ladybug to bring aid to the Unfettered Hai."
+	clearance
+	deadline 30
+	cargo "food (aid)" 121
+	destination "Firelode"
+	blocked "You need space for the agreed rush cargo of food to Unfettered space, and most use it as your flagship."
+	to offer
+		has "ship model: Ancient Ladybug"
+	on enter
+		dialog `As you take of from the planet you notice a message was left on the bridge, probably from the 'associates'. "You should use the Ladybug as your flagship and you should probably not bring any other ships. You'll understand why later." It occurs to you that you could also modify the ship.`
```

Comment:
I just want to inform the playr that they probably should change the ship layout if they have better tech
The fight later could be hard otherwise, or even very hard if you annoyed the unfettered a bunch before it